specification/storagetasks/resource-manager/Microsoft.StorageTasks/stable/2023-01-01/storagetasks.json

```diff
@@ -0,0 +1,903 @@
+            "schema": {
+              "$ref": "#/definitions/StorageTasksRunSummaryList"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "StorageTask": {
+      "type": "object",
+      "properties": {
+        "location": {
+          "type": "string",
+          "description": "Gets or sets the location of the resource. location property cannot be changes for successfully provisioned storage tasks."
+        },
+        "identity": {
+          "$ref": "#/definitions/StorageTaskIdentity",
+          "description": "The managed service identity of the resource."
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Gets or sets a list of key value pairs that describe the resource. These tags can be used in viewing and grouping this resource (across resource groups). A maximum of 15 tags can be provided for a resource. Each tag must have a key no greater in length than 128 characters and a value no greater in length than 256 characters."
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/StorageTaskProperties",
+          "description": "Properties of the storage task."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v1/types.json#/definitions/TrackedResource"
+        }
+      ],
+      "description": "Represents Storage Task."
+    },
+    "StorageTaskUpdateParameters": {
+      "type": "object",
+      "properties": {
+        "identity": {
+          "$ref": "#/definitions/StorageTaskIdentity",
+          "description": "The identity of the resource."
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Gets or sets a list of key value pairs that describe the resource. These tags can be used in viewing and grouping this resource (across resource groups). A maximum of 15 tags can be provided for a resource. Each tag must have a key no greater in length than 128 characters and a value no greater in length than 256 characters."
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/StorageTaskProperties",
+          "description": "Properties of the storage task."
+        }
+      },
+      "description": "Parameters of the storage task update request"
+    },
+    "StorageTaskIdentity": {
```

Comment:
>StorageTaskIdentity [](http://example.com/codeflow?start=5&length=19)

Please check v5 common types, as Identity is also available there, so reuse that instead of redefining it here.

Reply:
Made the change